lib/iex/lib/iex/helpers.ex

```diff
@@ -277,6 +279,106 @@ defmodule IEx.Helpers do
     end
   end
 
+  @module_info_keys [:module, :object_file, :source, :version, :compile_time, :compile_options]
+
+  @doc """
+  Prints out a list of all modules loaded and their BEAM file path.
+
+  """
+  def m() do
+    :code.all_loaded |>
+    Enum.sort |>
+    Enum.map(fn {module, path} ->m_display(module, path) end)
```

Comment:
Sorry, I'm so used to working in code bases that have linters that will deal with this for me. I've gotten in the habit of just writing what makes sense to me and then piping it into a linter that will 
convert what I've written to the standard for the project. 

Eventually I will submit a patch that actually meets your coding standards, I'll probably have to write the linter first, but I promise eventually I will get there.